### DISCUSSION TITLE
Who is the CEO of OpenAI?

### DISCUSSION THREAD

**Discussion by peymansf2000** [ParanoidUser/yolo#1259] [Galaxy Trivia] [answered]
who is it?! 
:)

**Comment by ParanoidUser** [comment, marked as answer]
Startup Yoda, Sam Altman! 🤖

**Reply by peymansf2000** [reply]
Correct but you have already the Galaxy Brain budget!

**Reply by peymansf2000** [reply]
Don't you post other trivia questions(for Galaxy Brain budget) today?

I'm ready for the challenge!

**Reply by ParanoidUser** [reply]
The next series will go out tomorrow. Stay tuned! 📻
